src/fparser/two/tests/utils/test_binaryopbase.py

```diff
@@ -124,26 +124,18 @@ def test_binaryopbase_match(pattern, string):
     assert result[2].string == "b"
 
 
-def test_binaryopbase_isadd():
-    '''Test the optional is_add argument to the BinaryOpBase match
-    method. This argument makes the associated pattern ignore the '+'
-    in a Real_Literal_Constant on the RHS of an expression. This
-    allows the expression to match the actual '+' operator for an
-    'add_op' pattern, e.g. 'a+3.0e+10' will match on the earlier '+'
-    rather than returning no-match. This option is a special case and
-    should probably be removed, see issue #281.
+def test_binaryopbase_addition():
```

Comment:
Well I disagree with this argument. It's like testing something that no longer exists. However, it is not that important so I'm not going to push you to remove it, it will just be confusing for anyone looking at it in the future.

Reply:
I guess you're going with the ignore me option :-p